redaxo/src/core/lib/list.php

```diff
@@ -76,6 +76,12 @@ class rex_list implements rex_url_provider_interface
     /** @psalm-var array<string, string|int> */
     private $formAttributes;
 
+    //  --------- Row Attributes
+    /** @psalm-var array<string, string|int>|callable  */
+    private $rowAttributes;
+    /** @var bool */
+    private $rowAttributesCallable;
```

Comment:
brauchen wir dieses attribut, oder können wir einfaher `is_callable($this->rowAttributes)` stattdessen verwenden?

Reply:
Klar, kann man auch in der jeweiligen Zeile abfrage, ist dann aber x mal. Hab das aus Performance-Gründen so gebaut. Alternative: den Code von fetchRowAttribute nach get schieben und is_callable vor der Schleife ausführen.

Reply:
ich kann mir nicht vorstellen dass wir einen messbaren impact bekommen bzgl. einem is_callable() check.
wenn du ein beispiel hast, wo es einen unterschied macht können wir es gerne beibehalten, allerdings vermute ich dass es hier ohne nutzen den code unnötig verkompliziert

Reply:
Da kennst Du dich um Längen besser aus als ich. Code ist geändert, fetchRowAttributes aufgelöst nach get(), $rowAttributesCallable entfernt. setRowAttributes() auch angepasst.